content/docs/command-reference/list.md

```diff
@@ -16,11 +16,12 @@ positional arguments:
 
 ## Description
 
-DVC, by effectively replacing data files, models, directories with `.dvc` files
-(`.dvc`), hides actual locations and names. This means that you don't see data
-files when you browse a <abbr>DVC repository</abbr> on Git hosting (e.g.
-GitHub), you just see the `dvc.yaml` and `.dvc` files. This makes it hard to
-navigate the project to find <abbr>data artifacts</abbr> for use with `dvc get`,
+DVC replaces data files, models, directories, etc. with small
+[metafiles](/doc/user-guide/dvc-files-and-directories#dvc-files-and-directories),
+and hides actual locations and names. Hence data files aren't visible when you
+browse a <abbr>DVC repository</abbr> on Git hosting (e.g. GitHub), you just see
+the `dvc.yaml` and `.dvc` files. This makes it difficult to find <abbr>data
+artifacts</abbr> while navigating in a project, for using with `dvc get`,
```

Comment:
I don't see who this addresses #1521 and it has some questionable language usage like "for using" among other phrases. The changes don't seem to bring any new info. anyway.